### DISCUSSION TITLE
Can one Github Actions self-hosted runner work for multiple repositories?

### DISCUSSION THREAD

**Discussion by DaHuoKolmostar** [community/community#25512] [Actions]
<p>Can one Github Actions self-hosted runner work for multiple repositories? Or is it going to be a feature in the future?</p>

**Comment by BrightRan** [comment, marked as answer]
<p>Currently, we have no methods to use the self-hosted runners across repositories, and also have no options to added self-hosted runners on Organization level.<br>
When installing the self-hosted runner, if enter the <strong>Organization URL</strong> , NOT a repository URL, as the server URL, it will return the <strong>404</strong> ( <strong>Not Found</strong> ) error.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a96b8dec2d11ed898594e7a361860009c29c7c37.png" data-download-href="/uploads/short-url/oaL7R5gdwbxJnxdE3a7EnYcoIpV.png?dl=1" title="test.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181091668-c180da5c-6377-428a-bc7d-33f17eb6d76b.png" alt="test.PNG" data-base62-sha1="oaL7R5gdwbxJnxdE3a7EnYcoIpV" width="690" height="451" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a96b8dec2d11ed898594e7a361860009c29c7c37_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">test.PNG</span><span class="informations">901×590 25.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
